SYCL/KernelAndProgram/built-in-kernel-ids.cpp

```diff
@@ -0,0 +1,35 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Can't we have this test in `intel/llvm:sycl/unittests`?

Reply:
I suppose so. Are you suggesting not performing a real device query at all and manually adding something into the program manager built-in kernel cache?

Reply:
It would be nice to check that  `sycl::device::get_info<sycl::info::device::built_in_kernel_ids>` returns expected value if `piDeviceGetInfo`  with `CL_​DEVICE_​BUILT_​IN_​KERNELS` returns non-empty list of kernels(we can emulate `piDeviceGetInfo`). And then check that an attempt to get a kernel bundle containing this kernel leads to exception.

Reply:
Added: intel/llvm#5046